src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final Random random = new Random();
+    private final int maxFigure;
```

Comment:
```suggestion
    private final int maxFigure = 4;
```
тобі нема сенсу приймати цей параметр ззовні класу так як в тебе він впирається в switch case в будь-якому випадку